src/senselab/utils/data_structures/model.py

```diff
@@ -35,10 +37,71 @@ def validate_path_or_uri(cls, value: Union[str, Path]) -> Union[str, Path]:
 
         return value
 
+
+class HFModel(SenselabModel):
```

Comment:
I am not sure I see the necessity for these model classes

Reply:
this way, we do check if the model actually exists on the hub